### ISSUE TITLE
Empty provider warning

### ISSUE THREAD

**Issue by art049** [milliHQ/terraform-aws-next-js#155]
Hi, I'm currently having a warning while running terraform apply/plan/delete.

Not sure it's a big deal but here is the bug report:
```
│ Warning: Empty provider configuration blocks are not required
│ 
│   on .terraform/modules/tf_next/provider.tf line 2:
│    2: provider "aws" {
│ 
│ Remove the aws.global_region provider block from module.tf_next. Add aws.global_region to the list of configuration_aliases for aws in required_providers to define the
│ provider configuration name.
```
Terraform version: v0.15.4 on linux_amd64
Terraform module version: 9.0.1
tf-next npm version: 0.7.0

I'd be happy to help on this if it's needed :smile:

**Comment by ofhouse** [comment]
Hi, thanks for the report.

Yes, I am aware of this warning.
It is raised by the way we forward the aws global provider internally to the module that handles the CloudFront distribution.

Terraform `v0.15` [introduced](https://www.terraform.io/upgrade-guides/0-15.html#alternative-provider-configurations-within-modules) a new way (`configuration_aliases`) to handle this and raises a deprecation warning when it detects a usage of the former syntax.

Fixing this would mean that we need to increase the minimum Terraform version for this module from `v0.13` to `v0.15` since the new syntax is not supported in older versions.

Given that `v0.15` was released in April I would wait at least until October (~6 months) to make this step to give users enough time to upgrade to a newer Terraform release fist.

**Comment by ofhouse** [comment]
This has now been fixed in [`v0.10.0`](https://github.com/milliHQ/terraform-aws-next-js/releases/tag/v0.10.0).
